components/scrollable-container/script.js

```diff
@@ -27,7 +27,19 @@ class ScrollableContainer extends BaseComponent {
 
     // eslint-disable-next-line require-jsdoc
     set automatic(value) {
-        this._automatic = value;
+        value ? this.setAttribute('automatic', 'automatic') : this.removeAttribute('automatic');
+    }
+
+    // eslint-disable-next-line require-jsdoc
+    get scrollPos() {
+        return this._scrollPos;
+    }
+
+    // eslint-disable-next-line require-jsdoc
```

Comment:
Document what is the expected value because I was misled how to use the setter :D

Reply:
Is it supposed to be number in pixels or number in % from 0 to 100? I tested it and seems to be %.

Reply:
It's percent. I will add a note. I also tested it with passing negative or above 100% values and the slider itself didn't allow it.

Reply:
And as a code comment would be nice also.